Translations/WinMerge/Dutch.po

```diff
@@ -2,25 +2,25 @@
 # Released under the "GNU General Public License"
 #
 # Translators:
-# * Ronald Schaap <ronald at schaap.demon.nl>
-# * Thomas De Rocker, 2022
+# Ronald Schaap <ronald at schaap.demon.nl>
```

Comment:
Removing the `*` at the beginning of this line prevents https://github.com/WinMerge/winmerge/blob/master/Translations/TranslationsStatus.md from being generated correctly.
So I added * in commit f5837c0.

Reply:
Oh, thank you for correcting it for me! I'll try to keep that in mind next time I upload a new version.